features/mesh/dataplanes/DataplanePolicies.feature

```diff
@@ -410,24 +417,6 @@ Feature: Dataplane policies
                 tags:
                   kuma.io/service: service-1
         """
-      And the URL "/meshes/default/dataplanes/dataplane-gateway_builtin-1/policies" responds with
-        """
-        body:
-          listeners:
-            - hosts:
-                - routes:
-                  - destinations:
-                      - tags:
-                          kuma.io/service: demo-app_kuma-demo_svc_5000
-                        policies:
-                          CircuitBreaker:
-                            name: circuit-breaker-1
-          policies:
-            TrafficLog:
-              name: traffic-log-1
-            TrafficTrace:
-              name: traffic-trace-1
-        """
```

Comment:
**Question**: Are you sure this can be removed? The endpoint seems to be called in this test and I’ve recently seen this test flake (though I haven’t looked into this).

(I’m aware that this only shows here because the PR isn’t rebased and that this mock was actually already removed)

Reply:
I merged master up into here so this has now gone from this PR.

The test was flaking before I removed this, and the PR I added to change this including an amend to fix the flake.

Nothing from this mock is being asserted on, apart from the mock having an amount of data).

Resolving this seeing as its not in this PR.